target/linux/mediatek/image/filogic.mk

```diff
@@ -96,6 +96,20 @@ define Build/cetron-header
 	rm $@.tmp
 endef
 
+define Build/append-teltonika-metadata
+	echo \
+			'{ \
+					"metadata_version": "1.1", \
+					"compat_version": "1.0", \
+					"version": "OpenWrt", \
+					"device_code": [".*"], \
+					"hwver": [".*"], \
+					"batch": [".*"], \
+					"serial": [".*"], \
+					"supported_devices":["teltonika,rutc"] \
+			}' | fwtool -I - $@
+endef
+
```

Comment:
There is similar functions in `target/linux/ipq40xx/image/generic.mk` could you please create a common one in `include/image-commands.mk` and make use of that in both targets. the ath79 target also has some code which looks similar, see `target/linux/ath79/image/generic.mk` .